src/templates/business.html

```diff
@@ -13,28 +14,159 @@ <h2>{{ business.name }}</h2>
+          <p><strong>Account Type:</strong> {{ request.account_type }}</p>
+          <p><strong>Account BIC:</strong> {{ request.account_bic }}</p>
+          <p><strong>Account Currency:</strong> {{ request.account_currency }}</p>
+          <p><strong>Status:</strong> {{ request.status }}</p>
+          <p><strong>IBAN:</strong> {{ request.iban }}</p>
+        </div>
+        <hr>
+        {% endfor %}
+      </div>
+    </div>
+  </div>
+
+  <div class="col-md-6">
+    <!-- Account Details -->
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4>Account</h4>
+      </div>
+      <div class="panel-body">
+        <table class="table">
+          <tr>
+            <td>Account ID</td>
+            <td>{{ business.account.id }}</td>
+          </tr>
+          <tr>
+            <td>IBAN</td>
+            <td>{{ business.account.iban }}</td>
+          </tr>
+          <tr>
+            <td>Type</td>
+            <td>{{ business.account.type }}</td>
+          </tr>
+          <tr>
+            <td>Balance</td>
+            <td>{{ business.account.balance.value / 100 }} €</td>
+          </tr>
+          <tr>
+            <td>Available Balance</td>
+            <td>{{ business.account.available_balance.value / 100 }} €</td>
+          </tr>
+          <tr>
+            <td>Locking Status</td>
+            <td>{{ business.account.locking_status }}</td>
+          </tr>
+          <tr>
+            <td>Status</td>
+            <td>{{ business.account.status }}</td>
+          </tr>
+        </table>
+      </div>
+    </div>
+
+    <!-- Beneficial Owners -->
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4>Beneficial Owners</h4>
+      </div>
+      <div class="panel-body">
+        {% for owner in business.beneficialOwners %}
+        <div class="form-group">
+          <p><strong>Owner ID:</strong> {{ owner.beneficial_owner_id }}</p>
+          <p><strong>Person ID:</strong> {{ owner.person_id }}</p>
+          <p><strong>Voting Share:</strong> {{ owner.voting_share }}%</p>
+          <p><strong>Relationship:</strong> {{ owner.relationship_to_business }}</p>
+        </div>
+        <hr>
+        {% endfor %}
+      </div>
+    </div>
+
+    <!-- Legal Representatives -->
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4>Legal Representatives</h4>
+      </div>
+      <div class="panel-body">
+        {% for representative in business.legalRepresentatives %}
+        <div class="form-group">
+          <p><strong>Representative ID:</strong> {{ representative.legal_representative_id }}</p>
+          <p><strong>Type of Representation:</strong> {{ representative.type_of_representation }}</p>
+          <p><strong>Power of Attorney Confirmed At:</strong> {{ representative.power_of_attorney_confirmed_at }}</p>
+        </div>
+        <hr>
+        {% endfor %}
+      </div>
+    </div>
```

Comment:
That's a little nitpicky, but wouldn't it make sense for the `LR's` to be above the `BO's`?